tests/requirements-310.txt

```
@@ -8,8 +8,8 @@ fastapi>=0.92.0
 flask>=2.3.2
 markupsafe>=2.1.0
 grpcio>=1.37.1
-google-cloud-pubsub<=2.1.0
-google-cloud-storage>=1.24.0
+google-cloud-pubsub>=2.23.0
```

You should keep the version number as before and only modify the operation. 
```suggestion
google-cloud-pubsub>=2.1.0
```

---

I'll change it but one question, it was <= 2.1.0 before and now we're changing it as >= 2.1.0, so we'll lose support for the < 2.1.0 . Is that okay?

---

I think we should make it <= 2.23.0 because our code is compatible up to 2.23 and may not be compatible with 2.24.0

---

Then, our CI pipeline, which checks the versions, will create a new currency ticket every time a new version is released.

Setting a lower-bound version will always get the latest version during our daily tests. If the tests pass, then we will be good for that version. If the tests break, we investigate the reason and fix our code if necessary, or we decide to set the upper-bound version.

Let's go for the lower-bound for now.